assets.js

```diff
@@ -1,5 +1,19 @@
 export const procgenAssetsBaseUrl = `https://webaverse.github.io/procgen-assets/`;
 export const urlSpecs = {
+  simplexMap: `${procgenAssetsBaseUrl}noise/simplex-noise.png`,

```

Comment:
This is the place to put assets which we will sprinkle into the world, not textures we will load. This should be in its own constant and this variable needs a better name.